torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -21,14 +23,28 @@ def __init__(
         self,
         data,
         *args,
+        edge_time=None,
```

Comment:
```suggestion
        edge_time: OptTensor = None,
```
But actually this is not used below except for checking if it is `None`, so maybe it should be a bool?

Reply:
Removed that check in `__init__`.